### PR TITLE
Amend ANGLE_multiview to allow for paused XFB.

### PR DESCRIPTION
**Pull request by null77**
Rationale: a paused XFB object is essentially invisible to the
rendering engine. The base ES spec says: "When transform feedback
is paused, no vertices are recorded." Moreover, many operations
that fail when an XFB is active are allowed when it is also paused.
This change makes the multiview behaviour consistent with the rest
of the spec.

PTAL @kbr @kainino0x @jdashg and feel free to CC any NVIDIA colleagues who might be impacted by this minor spec fix.

### CODE REVIEW

**Comment by null77** [comment]
For context: I'm trying to improve performance of draw call validation in ANGLE. This minor change would allow us to align the behaviour of ANGLE_multiview with other draw call validation.

**Comment by jdarpinian** [comment]
LGTM

File: extensions/WEBGL_multiview/extension.xml [hunk 1]
```diff
@@ -178,7 +178,7 @@ interface WEBGL_multiview {
       The error <code>INVALID_OPERATION</code> is generated by attempting to draw if the number of views in the current draw framebuffer is greater than one and the active program does not declare a number of views.
     </error>
     <error>
-      The error <code>INVALID_OPERATION</code> is generated by attempting to draw if the number of views in the current draw framebuffer is greater than one and transform feedback is active.
+      The error <code>INVALID_OPERATION</code> is generated by attempting to draw if the number of views in the current draw framebuffer is greater than one and transform feedback is active and unpaused.
```
[the remark this answers is not in the document]
Comment (null77) [reply extensions/WEBGL_multiview/extension.xml]: Thanks for the suggestion. Done.

**Comment by null77** [comment]
PTAL at amended wording.

**Comment by jdashg** [comment]
Thanks!

**Comment by jhelferty-nv** [comment]
LGTM too.

**Pull request merged**